charts/llmos-agents/templates/langflow/frontend-nginx-configmap.yaml

```
@@ -0,0 +1,66 @@
+
+            location = /index.html {
+                root /usr/share/nginx/html;
+                add_header Cache-Control "no-cache, no-store, must-revalidate";
+                etag on;
+            }
+
+            location /proxy/api/ {
+                proxy_pass ${BACKEND_URL};
+                #rewrite ^/proxy/api/(.*)$ /api/$1 break;
+            }
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)